endpoints/globus/endpoint_test.go

```diff
@@ -86,7 +86,8 @@ func setup() {
 	dtstest.EnableDebugLogging()
 
 	if _, ok := os.LookupEnv("DTS_GLOBUS_TEST_ENDPOINT"); !ok {
-		panic("DTS_GLOBUS_TEST_ENDPOINT environment variable must be set!")
+		print("DTS_GLOBUS_TEST_ENDPOINT environment variable not set. Skipping Globus unit tests.\n")
+		os.Exit(0)
```

Comment:
Will this bail out of the globus tests or out of all tests (`exit()` in python kills the whole process, not just a test set)?

Reply:
Good question. In Go, every package gets its own test process, so this just bails out of the Globus tests.